AnalogGridControl/InputAggregate.cs

```diff
@@ -137,9 +140,13 @@ public void UpdateInputs()
                 break;
 
               // Pitch in SE is inverted compared to how joysticks usually handle it
-              case GameAxis.TurnPitch: value = (value - 0.5f) * -40; break;
+              case GameAxis.TurnPitch: 
+              case GameAxis.CameraPitch:
+                value = (value - 0.5f) * -40; 
```

Comment:
Does the camera also require the expanded range as like turning? Or should it be the direct (i.e. not multiplied by 40) value?